app/presentation/components/canvas/basic_canvas.py

```diff
@@ -0,0 +1,58 @@
+from typing import List, Tuple
+
+from PyQt5 import QtGui, QtWidgets
+from PyQt5.QtGui import QPen, QPainter
+
+from app.consts.presentation_consts import (
+    CANVAS_WIDTH,
+    CANVAS_HEIGHT,
+    CANVAS_PEN_COLOR,
+    GRAY_COLOR,
+    GRID_WIDTH,
+    GRID_STEP,
+)
+
+
+class BasicCanvas(QtWidgets.QWidget):
```

Comment:
IMO as it is "abstract class" I should have used `ABC` class, but it is just programming mistake.